agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -1842,7 +1853,13 @@ async def get_app_variant_instance_by_id(variant_id: str) -> AppVariantDB:
     async with db_engine.get_session() as session:
         result = await session.execute(
             select(AppVariantDB)
-            .options(joinedload(AppVariantDB.base), joinedload(AppVariantDB.app))
+            .options(
+                joinedload(AppVariantDB.base),
+                joinedload(AppVariantDB.app),
+                joinedload(AppVariantDB.modified_by.of_type(UserDB)).load_only(
```

Comment:
I suggest getting user info based on user ID explicitly (different endpoint and explicit call).

Reply:
Applicable wherever this happens.